hooks/mari/collector.py

```diff
@@ -194,6 +200,13 @@ def collect_texture_elements(self, settings, parent_item):
             for channel in geo.channelList():
                 channel_name = channel.name()
 
+                # skip channels with invalid names
+                channel_template_key = self.sgtk.template_keys["channel"]
```

Comment:
I don't like the fact that I have the key name `channel` hardcoded here. But I suppose we can assume this, since we set this value in `fields["channel"]` as well.